src/main/java/com/mentalhealthapi/mentalhealthapi/service/IBlogService.java

```diff
@@ -12,6 +12,13 @@ public interface IBlogService {
      */
     List<Blog> fetchAll();
 
+    /**
+     * Fetch blog using an ID
+     * @param id a unique identifier for a blog
+     * @return blog or null if none
+     */
+    Blog fetchById(int id);
+
```

Comment:
I believe starting to wireframe more functions will help you realize what needs to be done, and also help you to better visualize the final product and what you may be missing. Getting an idea of every CRUD operation and fetch operations needed will help to eliminate tech debt quickly. 

I think the @data declaration is better suited to be above the class. This is preference but also seems to be how the instances I've seen of this are laid out.

Reply:
Good idea.

Reply:
This looks like an incomplete feature.
Are you using these anywhere?  I don't see them used.
If not in use, remove them.  Keep the codebase as small as possible to do the job you want to do.  Unused code is clutter, and bad documentation, which increases technical debt.
Only complete features, which meet the team's Definition of Done, should be merged to Main/Master.